src/assets/images/iconInstagram.svg

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="20px" height="20px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <g id="🗿" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-        <g id="icon-upload-sheet" transform="translate(-289.000000, -144.000000)" fill="#FFFFFF">
+    <g id="🗿" stroke="none" stroke-width="1" fill-rule="evenodd">
```

Comment:
worried this might break things, would like some guidance here cause having fill none makes it kinda hard to style.

Reply:
If you set the pre-existing fill to `#000` and use the `fill` prop on the svg component, it should work correctly. Idk if the same can be achieved with styles